src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.ts

```diff
@@ -1,100 +1,98 @@
-import { CHAIN_NAMESPACE, type ChainNamespace } from '@shapeshiftoss/caip'
-import { bn } from '@shapeshiftoss/chain-adapters'
+import { bchChainId, type ChainId } from '@shapeshiftoss/caip'
 import assert from 'assert'
 import BigNumber from 'bignumber.js'
 import type { Address } from 'viem'
-import { fromBaseUnit } from 'lib/math'
+import { bn } from 'lib/bignumber/bignumber'
+import { fromBaseUnit, toBaseUnit } from 'lib/math'
+import { isUtxoChainId } from 'lib/utils/utxo'
 import { subtractBasisPointAmount } from 'state/slices/tradeQuoteSlice/utils'
 
-import { UTXO_MAXIMUM_BYTES_LENGTH } from '../constants'
 import { MEMO_PART_DELIMITER } from './constants'
 import { poolToShortenedPool } from './longTailHelpers'
 
+// Allows us to convert a THORChain parser value back to its base unit
+// for testing purposes
+export const thorchainParserToBaseUnit = (thorchainParserValue: string) => {
+  if (thorchainParserValue.length < 3) {
+    throw new Error('Number too short to contain an exponent.')
+  }
+
+  const exponent = parseInt(thorchainParserValue.slice(-2), 10)
```

Comment:
I think it would be cool to export the `.slice(-2)` part to  an external function that would be a bit more descriptive because this number looks pretty magical from an external point of view

Reply:
Yeah we should absolutely comment that and permalink to the line on the etherscan contract view